src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1072,11 +1072,13 @@ private Task<BatchAccount> CreateBatchAccountAsync()
 
         private Task<IResourceGroup> CreateResourceGroupAsync()
         {
+            var tags = DelimitedTextToDictionary(configuration.Tags.Trim());
```

Comment:
The tags are comma separated, but DelimitedTextToDictionary defaults to newline, so this will not work as is.